doto-backend/src/routes/router.js

```diff
@@ -30,18 +30,18 @@ router.post('/users/add', function (req, res) {
     user.save(function(err){
```

Comment:
we should really have a separate file for handling user data compared with task data. @allenwang968 im assuming the authentication people are already dealing with this endpoint so this could be deleted?